AC-2981: publish ban_customer (#1663)
livechat/livechat-public-docs: src/pages/management/configuration-api/v3.7/index.mdx

<Text>

### Ban Customer

> Bans a customer for a given period of time. While banned, the customer cannot connect to or interact with the [Customer Chat API](/messaging/customer-chat-api).

Reviewer: ```suggestion
Bans a customer for a specified duration. While banned, the customer cannot connect to or interact with the [Customer Chat API](/messaging/customer-chat-api) within your organization.
```
